app/models/api_docs/service.rb

```diff
@@ -38,6 +44,34 @@ class ApiDocs::Service < ApplicationRecord
 
   validates :base_path, non_localhost: true
 
+  def owner_account
+    return unless owner
+
+    owner_type == 'Account' ? owner : owner.account
+  end
+
+  # [Deprecated][TODO] remove with service_id
+  def service=(service_to_assign)
+    self.owner = self.owner_type = self.owner_id = nil
+    super(service_to_assign)
+  end
```

Comment:
This won't be needed if we do https://github.com/3scale/porta/pull/2116/files?file-filters%5B%5D=.rb#r489444922

Reply:
... and also https://github.com/3scale/porta/pull/2116/files?file-filters%5B%5D=.rb#r489448808